index.html

```diff
@@ -2,9 +2,181 @@
 <html lang="en">
 <head>
 	<meta charset="UTF-8">
-	<title>Title</title>
+	<link rel="stylesheet" href="styles.css">
+	<title>Rekupe - Digital Agency in LA</title>
 </head>
 <body>
+<header class="header__outer-container">
+	<div class="header__inner-container">
+		<nav class="navigation">
+			<ul class="navigation__list">
+				<li class="navigation__list-items"><a class="navigation___link" href="#main__content-section">case studies</a></li>
```

Comment:
Let op dat je niet te veel classes aanmaakt voor elementen. Dit konden we ook oplossen door simpelweg alleen de a tag in het CSS file aan te spreken om voor alle a tags ervoor te zorgen dat er geen text-decoration is en een alternatieve kleur.